File: array/array.cs
// array.cs
using System;

class Program {
    static void Main() {
        int[] arr = { 1, 2, 3, 4, 5 };
        Console.Write("Array elements are: ");
        foreach (int i in arr) {
            Console.Write(i + " ");
        }
        Console.WriteLine();
    }
}

File: array/array.cpp
// array.cpp
#include <iostream>

int main() {
    int arr[5] = {1, 2, 3, 4, 5};

    std::cout << "Array elements are: ";
    for (int i = 0; i < 5; i++) {
        std::cout << arr[i] << " ";
    }
    std::cout << std::endl;

    return 0;
}
